Makefile

```diff
@@ -35,37 +35,51 @@ verify: ## Verify code. Includes dependencies, linting, formatting, etc
 	go fmt ./...
 	golangci-lint run
 
+helm: ## Generate Helm Chart
+	cp $(RELEASE_MANIFEST) charts/karpenter/templates
+	yq w -i charts/karpenter/Chart.yaml version $(RELEASE_VERSION)
+	cd charts; helm package karpenter; helm repo index .
+
 codegen: ## Generate code. Must be run if changes are made to ./pkg/apis/...
 	controller-gen \
 		object:headerFile="hack/boilerplate.go.txt" \
 		webhook \
 		crd:trivialVersions=false \
 		paths="./pkg/..." \
-		output:crd:artifacts:config=config/crd/bases \
-		output:webhook:artifacts:config=config/webhook
+		output:crd:artifacts:config=config \
+		output:webhook:artifacts:config=config
 
 	./hack/boilerplate.sh
 
+	mv config/provisioning.karpenter.sh_provisioners.yaml config/provisioner-crd.yaml
+	mv config/manifests.yaml config/webhooks.yaml
+
+	# Fixup Webhook code generation. controller-gen assumes using kustomize; we do this instead
+	yq w -i -d0 ./config/webhooks.yaml 'metadata.name' karpenter-mutating-webhook-configuration
+	yq w -i -d0 ./config/webhooks.yaml 'metadata.annotations[cert-manager.io/inject-ca-from]' karpenter/karpenter-serving-cert
+	yq w -i -d0 ./config/webhooks.yaml 'webhooks[0].clientConfig.service.name' karpenter-webhook-service
+	yq w -i -d0 ./config/webhooks.yaml 'webhooks[0].clientConfig.service.namespace' karpenter
+	yq d -i -d0 ./config/webhooks.yaml 'webhooks[0].admissionReviewVersions[0]'
+	yq w -i -d1 ./config/webhooks.yaml 'metadata.name' karpenter-validating-webhook-configuration
+	yq w -i -d1 ./config/webhooks.yaml 'metadata.annotations[cert-manager.io/inject-ca-from]' karpenter/karpenter-serving-cert
+	yq w -i -d1 ./config/webhooks.yaml 'webhooks[0].clientConfig.service.name' karpenter-webhook-service
+	yq w -i -d1 ./config/webhooks.yaml 'webhooks[0].clientConfig.service.namespace' karpenter
+	yq d -i -d1 ./config/webhooks.yaml 'webhooks[0].admissionReviewVersions[0]'
```

Comment:
I do wonder if it would be worth moving this into a script under hack? It just seems like something where we ideally wouldn't keep changing the Makefile whenever it changes (I'd find it a bit easier to think about ...) Not a big deal, can just ignore for now, etc.